.github/workflows/build.yml

```diff
@@ -1,58 +1,74 @@
 # Build process
+# Generate go and python artifacts using 3.9
+# Test generated go artifacts using latest go
+# Test generate python artifacts using 2.7 and 3.9
+
 name: CICD
 
 on:
   - push
   - workflow_dispatch
 
 jobs:
-  generate:
-    runs-on: ubuntu-20.04
+  build:
+    runs-on: ubuntu-latest
     strategy:
       max-parallel: 1
       matrix:
-        python-version: [2.7]
+        python-version: [3.9]
     steps:
       - name: Checkout source
         uses: actions/checkout@v2
         with:
           ref: ${{ github.head_ref }}
-      - name: Setup Go for CI
-        uses: actions/setup-go@v2
+      - uses: actions/setup-go@v2
         with:
-          go-version: '1.17.3'
-      - name: Setup Go and protoc
+          go-version: "^1.17.1"
+      - name: Setup protoc
         run: |
-          python do.py setup_ext
+          python do.py get_protoc
```

Comment:
use `python setup_ext` only (since it checks for incorrect platform)

Reply:
change setup_ext